.github/workflows/destroy-vcluster.yml

```diff
@@ -0,0 +1,40 @@
+name: "Destroy vcluster"
+
+on:
+  pull_request:
+    types:
+      - closed
+  push:
+    branches:
+      - master
```

Comment:
will this make it run twice (on pr close and merge to master)?

Reply:
I messed this up, I meant to only run this on PR close